app/src/main/java/com/example/capstone/HomeFragment.java

```diff
@@ -84,13 +107,71 @@ public void onClick(View view) {
             @Override
             public void onClick(View v) {
                 Log.i(TAG, "onClick start session button");
-                // changes here
                 ((HomescreenActivity)getActivity()).replaceFragment(R.id.homescreen, StartSessionLogisticsFragment.newInstance(""));
             }
         });
 
+        adapter = new JoinedSessionsAdapter(getActivity(), getAllJoinedStudySessions());
+
+        rvJoinedStudySessions.setAdapter(adapter);
+        rvJoinedStudySessions.setLayoutManager(new LinearLayoutManager(getActivity()));
+
+        queryEachJoinedStudySessions();
+    }
+
+    private List<String> getListFromJsonArray(JSONArray jsonArray) throws JSONException {
+        List<String> list = new ArrayList<String>();
+        for(int i = 0; i < jsonArray.length(); i++){
+            list.add(jsonArray.getString(i));
+        }
```

Comment:
You can look into the `list.map()` function for a one-line way to do this in the future